src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,22 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SortCommand extends Command {
```

Comment:
Would be good to have proper documentation before the class implementation to have a overview of what the class will be doing.